src/main/java/org/broadinstitute/hellbender/tools/exome/AnnotateTargets.java

```diff
@@ -0,0 +1,252 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import htsjdk.tribble.bed.BEDFeature;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ExomeAnalysisProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Nucleotide;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Tool to annotate targets.
+ * <p>
+ *     This tools adds new columns to an existing target table with various target {@link TargetAnnotation annotations}.
+ * </p>
+ * <p>
+ *     Each annotation may require some source of information. For example {@link TargetAnnotation#GC_BIAS}
+ *     requires a reference, {@link TargetAnnotation#REPEAT_FRACTION} requires a list of repeat intervals.
+ * </p>
+ * <p>
+ *     Only annotation for which all data source dependencies are satisfied will be added to the ones present in the input
+ *     target table into the output target table.
+ * </p>
+ * <p>
+ *     There must be at least one annotation whose dependencies are satisfied, otherwise the
+ *     tool will return an error.
+ * </p>
+ * <p>
+ *     These are the currently supported annotations and their dependencies:
+ *     <dl>
+ *         <dt>{@link TargetAnnotation#GC_BIAS GC_BIAS}</dt>
```

Comment:
This annotation should be called "GC_CONTENT" or "PERCENT_GC" etc, not  GC bias.  There's only bias if changes in content lead to changes in coverage.